src/components/match-card.tsx

```diff
@@ -5,30 +5,35 @@ import Card from '@/components/card'
 import { css } from '@linaria/core'
 import { memo } from '@/utils/memo'
 import clsx from 'clsx'
+import Icon from './icon'
+import { mdiCheckCircle } from '@mdi/js'
 
 interface MatchCardProps {
   match: {
     key: string
     redAlliance: string[]
     blueAlliance: string[]
     time?: Date
+    redScore?: number | undefined
   }
   key?: string | number
   eventKey: string
   link?: boolean
   class?: string
+  checkmark?: boolean
```

Comment:
@varCepheid It looks like the only place where this is _not_ set is on event-match. Is there a reason to not show it there? It seems like it might simplify it to always show the checkmark on played match cards, regardless of where in the UI you are. What do you think?